docs/semgrep-supply-chain/overview.md

```diff
@@ -40,6 +40,8 @@ Semgrep Supply Chain generates a **finding** when it detects a match. If the dep
 
 A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
 
+A finding is **undetermined** if Semgrep detects a match for the dependency in the vulnerable version range, but does not scan for the reachability of this vulnerability. This is most common with older or lower severity vulnerabilities.
```

Comment:
I think we can also just say older and lower severity vulnerabilities don't have reachability rules? I feel like some users might interpret "does not scan for the reachability of this vulnerability" as a parsing/code issue

Reply:
Would you prefer:

A finding is **undetermined** if Semgrep detects a match for the dependency in the vulnerable version range, but does not have a reachability rule for this vulnerability.

or something else? Open to suggestions!

Reply:
@andy-r2c got a sec to take a look at this alternative?

Reply:
"A finding is undetermined if Semgrep detects a match for the dependency in the vulnerable version range, but does not have a reachability rule for this vulnerability."

works perfectly!